### DISCUSSION TITLE
Transport controls and iPad formatting

### DISCUSSION THREAD

**Discussion by abwino** [punxaphil/custom-sonos-card#63] [Q&A] [answered]
Hi,

Is it possible to make the play/pause button bigger please? It seems tiny on an iPad screen. On that note, Is it possible to get the card to recognise a tablet? At the moment, the iPad view seems to mirror my desktop view which makes everything too small on the iPad. Sorry if I have missed something here. The mobile settings don't seem to affect the iPad view.

Thanks very much!

Andrew.

**Comment by punxaphil** [comment, marked as answer]
Have you tried playing around with layout?
see `layout:` here https://github.com/johanfrick/custom-sonos-card#usage

**Reply by abwino** [reply]
Yes, I've tried all sorts of combinations, but it seems that if I have a display that fills up the iPad screen nicely it looks terrible on my desktop. Does the card actually recognise tablets in the same way as a mobile?

**Reply by punxaphil** [reply]
As long as you don’t have the exact same resolution on desktop and tablet you can play around with the mobile threshold

**Reply by punxaphil** [reply]
Or simply use two different dashboards

**Comment by abwino** [comment]
Oh! That’s a great idea, I would never have thought of that! Thanks very much.

Kindest regards,

Andrew.


From: Johan Frick ***@***.***>
Date: Sunday, 8 May 2022 at 20:57
To: johanfrick/custom-sonos-card ***@***.***>
Cc: Andrew Brown ***@***.***>, Author ***@***.***>
Subject: Re: [johanfrick/custom-sonos-card] Transport controls and iPad formatting (Discussion #63)

Or simply use two different dashboards

**Comment by BigFriendlyDuck** [comment]
Thank for a great HACS-card, Johan! It's by far the most used application on my HA-dashboard.

Per your suggestion, I've tried adjusting different settings, both CSS and other, and have come up with a descent dash.

But it would be awesome if it was possible to arrange the "modules" (mediaBrowser, mediaItem, groups, etc.) in two columns. (I don't think it's possible per now, but I might be wrong?)

For example I'd like to have the player and the grouping-module in one column and groups and mediaItem in one column.
And the possibillity to change the size of the pause/play/other buttons it would be awesome, as they are a tad to small on my iPad.

**Reply by punxaphil** [reply]
How about this?
```
type: custom:custom-sonos-card
layout:
  groups:
    width: 50%
  players:
    width: 50%
  mediaBrowser:
    width: 50%
  mediaItem:
    width: 25%
styles:
  groups:
    order: 2
  content:
    justifyContent: flex-start

```
